.gitignore

```diff
@@ -1,28 +1,28 @@
-*.py[co]
-.DS_Store
-_build
-build
-dist
-*.egg-info
-*~
 *.bak
-*.swp
-plots/
-log.txt
-.ipynb_checkpoints/
-.cache
-.tox
-.vagrant
-wintest.sh
-Vagrantfile
 *.class
+*.dist-info/
+*.egg-info
 *.eggs/
+*.py[co]
+*.swp
+*~
+.DS_Store
+.cache
 .coverage
-htmlcov
-*.dist-info/
-.vscode
 .idea
+.ipynb_checkpoints/
 .pytest_cache/
-docs/examples/**/*.py  # For .ipynb examples converted to .py
-.ci/*.sh
+.tox
+.vagrant
+.vscode
+Vagrantfile
+_build
+bones-scripts/
+build
+dist
 docs/.doctrees
+docs/examples/**/*.py  # For .ipynb examples converted to .py
+htmlcov
+log.txt
+plots/
+wintest.sh
```

Comment:
Doesn't really matter, but `wintest.sh` seems like an oddly specific thing to have in our gitignore.

Reply:
I believe that this was for a very old way I tried to do Windows testing using Vagrant. Definitely not relevant anymore, haha